### DISCUSSION TITLE
Traduction/ Translation/Traducción

### DISCUSSION THREAD

**Discussion by Moigaston** [chamilo/chamilo-lms#5606] [General]
Comment participer à la traduction de Chamilo en allemand?

**Comment by lonesomewalker** [comment]
In the past it was translate.chamilo.org, but maybe you can also do a pull request...

**Comment by Moigaston** [comment]
Cela fait plusieurs fois (déjà sur l'ancien forum !) que je propose de participer à la traduction de Chamilo en allemand : je trouve dommage qu'on ne m'ait pas répondu ni indiqué comment y participer. D'autant plus qu'il reste beaucoup à faire !

I have offered several times (even on the old forum!) to help with the translation of Chamilo into German: I find it unfortunate that no one has replied to me or explained how to participate. Especially since there is still a lot to do!

Ya he propuesto varias veces (¡incluso en el foro antiguo!) participar en la traducción de Chamilo al alemán: me parece una pena que no me hayan respondido ni indicado cómo participar. ¡Sobre todo porque todavía queda mucho por hacer!

**Reply by ywarnier** [reply]
Bonjour @Moigaston 

I'll write in English for better follow up by anybody else as this applies to many languages and potential contributors.
First of all, thank you for offering and sorry you weren't answered to earlier. I don't check the forum very often and this year the dev team is extra busy working in parallel on Chamilo 1.11.28 and Chamilo 2.0. We also switched from the old forum to here, which possibly increased the difficulty for us to find your question and provide you with an answer. This is just human error, not a voluntary dismissal of your contribution.

Second, we like translators to be native (or near-native) in their *destination* language, in particular for West-European languages, which are the ones that are the most used with Chamilo (so an extra level of care has to be adopted). I hope this is the case for you and German (giving you seem to mostly work in French).

Third, you come here at a complex time because we are between two major versions with drastically different translation methods. Where Chamilo 1.11.x was translated through a custom system developed in-house (https://translate.chamilo.org) and Chamilo 2.x is using Gettext and has an online translation system that is not yet opened to the public (because we are still adding many terms).

I will suppose you want to help perfect the 1.11.x translation in view of version 1.11.28 soon to be released. In this case, please register on https://translate.chamilo.org, then indicate (in "Help translate Chamilo!") from which language to which language you want to translate and let me know (here, for example) when it's done. I will then contact you privately to check your background and see if we can accept your contribution, and if everything checks out, you'll be able to contribute to v1.11.28.

**Comment by Kaneda-1** [comment]
Hi @Moigaston 

Contact the Chamilo Association at
https://chamilo.org/en/contact/ - info@chamilo.org

**Comment by Moigaston** [comment]
Bonjour @ywarnier,
Thank you for your prompt reply. French is indeed my mother tongue, but as for my German skills, I have been living in Germany for over thirty years. I completed my studies there, which I had started in France, and I also wrote a master's thesis in philosophy at the University of Hamburg, where I reside. Finally, I am a high school teacher, and I teach French (of course!), history, philosophy, and... German as a foreign language. I therefore use Chamilo in German ([img]https://i.postimg.cc/6QhbN8BS/notre-site-chamilo.png[/img][/url]). In 2020, I managed to convince the Moodle administrator at my school to switch to Chamilo. Since then, we have both been administrators and regularly train our colleagues on how to use Chamilo. Moreover, we are so convinced by Chamilo that we have recently decided to promote it in the German educational system, where only Moodle is known. These are all the reasons why I, along with my colleague, would like to contribute to translating Chamilo (1.11.28 and 2.0) into German.

**Reply by ywarnier** [reply]
@Moigaston 
That seems all good to me. Thanks for the details.
I see your registration on translate.chamilo.org, but your registered to translate from English to Asturian. Could you ask again but for English to German (or French to German), please?

**Comment by Moigaston** [comment]
Bonjour @ywarnier
I tried, but the German language ("Deutsch") doesn't appear in the list: [url=https://postimages.org/][img]https://i.postimg.cc/Ls9SQKWc/Translation.png[/img][/url]

**Reply by ywarnier** [reply]
Apparently that was because you did apply for English -> German some time ago, so you couldn't pick it anymore.
I just removed your English->Asturian request and authorized your English->German request. Let me know if you have any question, and thank you for your (coming) help!
Please note that we should be releasing 1.11.28 next week (9/10), so if you want to put a few translations in, these will be included straight away.

**Comment by Moigaston** [comment]
`I just (...) authorized your English->German request.`
It doesn't work, unfortunately:[url=https://postimages.org/][img]https://i.postimg.cc/KYFNKLv2/Translation1.png[/img][/url][url=https://postimages.org/fr/]free image upload[/url] German is still not on the list.
On https://translate.chamilo.org/, it says: `your destination language will be marked in green`, but that is not the case either.
